.changes/config.json

```diff
@@ -144,9 +144,66 @@
       "manager": "javascript",
       "assets": [
         {
-          "path": "./tooling/api/tauri-apps-api-${ pkgFile.version }.tgz",
+          "path": "./tooling/api/dist/tauri-apps-api-${ pkgFile.version }.tgz",
```

Comment:
`npm pack` is going to place the tar here, but I think when we `npm publish`, that this will be included. We probably want to move it or have npm ignore it.